buildlib/tools/builds.sh

```diff
@@ -377,6 +377,31 @@ build_fuse() {
 	fi
 }
 
+#
+# Build with static library
+#
+build_static() {
+	${WORKSPACE}/contrib/configure-devel --prefix=$ucx_inst
+	$MAKEP
+	$MAKEP install
+
+	cd test/apps/uct_info
+
+	$MAKE
+
+	if ./uct_info && ./uct_info_static
```

Comment:
1. could just run uct_info and uct_info_static (since we use bash -Ee)
2. can check list of transports in the output of these commands?
- they should be the same
- they should contain builtin transports : TCP, self, posix

Reply:
> 1. could just run uct_info and uct_info_static (since we use bash -Ee)

done

> 2. can check list of transports in the output of these commands?
> 
> * they should be the same
> * they should contain builtin transports : TCP, self, posix

list of transports is tested by script called from ```make test``` below. comparing of list of transports will always fail because we have not covered static links for all transports (for now), but later it may be done